src/validators/format.js

```diff
@@ -0,0 +1,20 @@
+const checker = require("../checker");
+const err = require("../errorCodes");
+
+function format(value, expression) {
+  let results = null
+  if (checker.isEmpty(value)) {
+    return null;
+  }
+
+  const result = expression.test(value);
+  if (result === false || !checker.isString(value))
```

Comment:
I guessing the `isString` validation here is regarding the `isEmpty` bug reported, right? If so, could you move it at the begging of the method as a guar clause?

Reply:
I made a change and removed this mandatory value from the string. I think we can gain from the flexibility that javascript gives us and let the regex say whether the expression is valid or not